src/Redux/Delegates.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace Redux 
+{
+    /// <summary>
+    ///     Represents a method that dispatches an action.
+    /// </summary>
+    /// <param name="action">
+    ///     The action to dispatch.
+    /// </param>
+    /// <returns>
+    ///     Varies
+    /// </returns>
+    public delegate dynamic Dispatcher(object action);
```

Comment:
to be honest I don't know the difference.  I had it dynamic at first and then changed it since depending on the middleware it could be different.  What's the difference?